addons/binding/org.openhab.binding.autelis/src/main/java/org/openhab/binding/autelis/handler/AutelisHandler.java

```diff
@@ -134,7 +134,6 @@ public AutelisHandler(Thing thing) {
 
     @Override
     public void initialize() {
-        super.initialize();
```

Comment:
Yes because the line below calls `configure()` which sets the thing to offline immediately anyway. See [here](https://github.com/openhab/openhab2-addons/blob/master/addons/binding/org.openhab.binding.autelis/src/main/java/org/openhab/binding/autelis/handler/AutelisHandler.java#L232)